include/tvm/runtime/packed_func.h

```diff
@@ -470,6 +472,12 @@ class TVMPODValue_ {
     TVM_CHECK_TYPE_CODE(type_code_, kNDArrayContainer);
     return NDArray(static_cast<NDArray::Container*>(value_.v_handle));
   }
+  operator Object() const {
+    if (type_code_ == kNull) return Object();
+    TVM_CHECK_TYPE_CODE(type_code_, kObject);
+    auto ptr = ObjectPtr<ObjectCell>(static_cast<ObjectCell*>(value_.v_handle));
```

Comment:
why not directly allow construction from the intrusive ObjectCell ptr like NDArray?

Reply:
@tqchen Did you mean `return Object(static_cast<ObjectCell*>(value_.v_handle));`?

Reply:
Yes, if object support similar function as NDArray